zerver/tests/test_messages.py

```diff
@@ -4531,8 +4531,8 @@ def assert_last_um_content(user: UserProfile, content: str, negate: bool=False)
         soft_deactivated_user_msg_count = len(get_user_messages(long_term_idle_user))
         message = 'Testing test_alert_word'
         send_stream_message(message)
-        assert_last_um_content(long_term_idle_user, message)
-        assert_um_count(long_term_idle_user, soft_deactivated_user_msg_count + 1)
+        assert_last_um_content(long_term_idle_user, message, negate=True)
+        assert_um_count(long_term_idle_user, soft_deactivated_user_msg_count)
```

Comment:
What's the story with these changes?

Reply:
I missed out the deactivated users part.